openslides/assignments/static/templates/assignments/assignment-detail.html

```diff
@@ -77,10 +77,10 @@ <h3 translate>Phase</h3>
 <div class="details">
   <div ng-if="assignment.description">
     <h3 translate>Description</h3>
-    <div class="white-space-pre-line">{{ assignment.description }}</div>
+    <div ng-bind-html="assignment.description | trusted"></div>
   </div>
 
-  <div ng-if="assignment.phase !== 2">
+  <div ng-if="assignment.phase !== 2" style="clear: both">
```

Comment:
Bootstrap has a class `clearfix` for this. Same for assignemt_slide.html

Reply:
Done